rskj-core/src/main/java/co/rsk/peg/StateForProposedFederator.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * This file is part of RskJ
+ * Copyright (C) 2024 RSK Labs Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package co.rsk.peg;
+
+import co.rsk.bitcoinj.core.BtcTransaction;
+import co.rsk.crypto.Keccak256;
+import java.util.Map;
+import org.ethereum.util.RLP;
+
+public class StateForProposedFederator {
+
+    private final Map.Entry<Keccak256, BtcTransaction> rskTxWaitingForSignatures;
```

Comment:
Since this will be for a very specific use case, what do you think about calling it `svpSpendTxWaitingForSignatures`

Reply:
Should we also change the other one to `pegoutsWaitingForSignatures`? Or too specific?

Reply:
That one can be pegouts, migrations, fund tx. This one is very specific

Reply:
Updated